lab_6_pipeline/pipeline.py

```diff
@@ -55,26 +110,48 @@ def __init__(self, text: str):
         """
         Initializes ConlluToken
         """
+        self._text = text
+        self.position = 0
+        self._morphological_parameters = MorphologicalTokenDTO()
 
     def set_morphological_parameters(self, parameters: MorphologicalTokenDTO) -> None:
         """
         Stores the morphological parameters
         """
+        self._morphological_parameters = parameters
+
+    def set_position(self, position: int) -> None:
+        self.position = position
 
     def get_morphological_parameters(self) -> MorphologicalTokenDTO:
         """
         Returns morphological parameters from ConlluToken
         """
+        return self._morphological_parameters
 
     def get_conllu_text(self, include_morphological_tags: bool) -> str:
         """
         String representation of the token for conllu files
         """
+        position = str(self.position)
+        text = self._text
+        lemma = self._morphological_parameters.lemma
+        pos = self._morphological_parameters.pos
+        xpos = '_'
+        feats = self._morphological_parameters.tags \
+            if include_morphological_tags and self._morphological_parameters.tags else '_'
+        head = '0'
+        deprel = 'root'
+        deps = '_'
+        misc = '_'
+        return '\t'.join([position, text, lemma, pos,
+                          xpos, feats, head, deprel, deps, misc])
 
     def get_cleaned(self) -> str:
         """
         Returns lowercase original form of a token
         """
+        return self._text.lower().translate(str.maketrans('', '', string.punctuation))
```

Comment:
1. very complicated, use regex for replacement of punctuations
2. do we lose underscores in this manner?